src/colors.rs

```diff
@@ -22,6 +22,12 @@ use termcolor::BufferWriter;
 #[cfg(windows)]
 use termcolor::ColorChoice;
 
+static FORCE_COLOR: Lazy<bool> = Lazy::new(|| {
+  std::env::var_os("FORCE_COLOR")
+    .map(|v| !v.is_empty())
+    .unwrap_or(false)
+});
+
 static USE_COLOR: Lazy<AtomicBool> = Lazy::new(|| {
   #[cfg(wasm)]
```

Comment:
I wonder if we should do something similar for wasm for FORCE_COLOR? Maybe for now just return `false` when wasm in `FORCE_COLOR` until someone needs it?

Reply:
The comment above says they are avoiding checking env vars, so I don't think we should use it